src/index.html

```diff
@@ -5,9 +5,212 @@
+              Working directly with developers - so have more useful information about the<br>
+              profitable deals.
+            </p>
+            <p>
+              <strong>We will never be late!<br>
+              Guaranteed secure transaction - 100%.</strong>
+            </p>
+          </div>
+        </div>
+      </section>
+      <section class="section">
+        <a name="gallery"></a>
+        <h2 class="heading section__heading">Gallery
+          <span class="heading__crown"></span>
+        </h2>
+        <div class="gallery">
+          <div class="photos photos_wide_left">
+            <img src="images/photos/1.jpg" alt="" class="photos__img">
+            <img src="images/photos/2.jpg" alt="" class="photos__img">
+          </div>
+          <p class="gallery__title">Marina Palms /
+            <a href="#" class="link">North Miami Beach, FL 33162</a>
+          </p>
+          <div class="photos">
+            <img src="images/photos/3.jpg" alt="" class="photos__img">
+            <img src="images/photos/4.jpg" alt="" class="photos__img">
+          </div>
+          <p class="gallery__title">Edition Resident /
+            <a href="#" class="link">Miami Beach, FL 33139</a>
+          </p>
+          <div class="photos photos_wide_right">
+            <img src="images/photos/5.jpg" alt="" class="photos__img">
+            <img src="images/photos/6.jpg" alt="" class="photos__img">
+          </div>
+          <p class="gallery__title">Faena House /
+            <a href="#" class="link">Miami Beach, FL 33140</a>
+          </p>
+          <button class="button gallery__button">View all</button>
+        </div>
+      </section>
+      <section class="section">
+        <a name="consultation"></a>
+        <h2 class="heading section__heading">Consultation
+          <span class="heading__crown"></span>
+        </h2>
+        <div class="consultation">
+          <div class="consultation__text">
+            <h3 class="consultation__heading">How long have<br> do yourself a gift?</h3>
+            <p>The procedural change mezzo forte starts izoritmichesky pickup<br>
+              at these moments stop LA Mazel and VA Tsukkerman in his<br>
+              "Analysis of musical works." Pointillism, which originated in the<br>
+              music of the early twentieth century, microforms, found a distant<br>
+              historical.</p>
+          </div>
+          <form class="form consultation__form">
+            <label class="form__item">
+              <input
+                type="text"
+                name="name"
+                placeholder="Name"
+                class="input form__input">
+            </label class="form__item">
+            <label class="form__item">
+              <input
+                type="text"
+                name="phone"
+                placeholder="Phone"
+                class="input form__input">
```

Comment:
закрывающая скобка также переносится
```suggestion
                class="input form__input"
              >
```